src/main.rs

```diff
@@ -29,20 +25,13 @@ fn main() {
     pretty_env_logger::init();
 
     // Turn our state into a Filter so we can combine it easily later
-    let bus_cache = TimedString {
-        time: Instant::now() - Duration::from_secs(30),
-        url: String::from("http://developer.itsmarta.com/BRDRestService/RestBusRealTimeService/GetAllBus"),
-        text: Bytes::from(&b"<unused>"[..])
-    };
+    let bus_cache = TimedString::new("http://developer.itsmarta.com/BRDRestService/RestBusRealTimeService/GetAllBus",
+                                     Duration::from_secs(10));
```

Comment:
default `TimedString` constructor takes just a URL and a duration for the cache to be valid.